test/AutoRest.TestServerLowLevel.Tests/url-path-items.cs

```diff
@@ -12,55 +12,76 @@ public class UrlPathItemsTests : TestServerLowLevelTestBase
     {
         [Test]
         public Task UrlPathItemGetAll() => TestStatus(async (host) =>
-            await new PathItemsClient(globalStringPath: "globalStringPath",
-                    globalStringQuery: "globalStringQuery",
+        {
+            var options = new AutoRestUrlTestServiceClientOptions()
```

Comment:
If `globalStringQuery` is optional, eventually we can create a client without this parameter. But now, it seems that we already need to set this parameter in ClientOptions, that is not correct.

Reply:
You want test `var options = new AutoRestUrlTestServiceClientOptions()`? 

This is not an e2e test, so above statement just shows we could create a class without initializing the optional properties.